[esx]/es_extended/client/main.lua

```diff
@@ -64,53 +64,58 @@ AddEventHandler('esx:playerLoaded', function(xPlayer, isNew, skin)
 			local RemoveAllPickupsOfType = RemoveAllPickupsOfType
 			local HideHudComponentThisFrame = HideHudComponentThisFrame
 			local PlayerId = PlayerId()
+			local ESXPlayerId = ESX.PlayerData.ped
```

Comment:
Why is this called 'ESXPlayerId' and not 'playerPed' or something similar?

Reply:
Fixed @Gittified